test/unit/Dredd-test.js

```diff
@@ -116,20 +124,31 @@ describe('Dredd class', () => {
 
       it('should load file contents on paths to config', done => dredd.run((error) => {
         if (error) { return done(error); }
-        assert.isObject(dredd.configuration.data);
-        assert.property(dredd.configuration.data, './test/fixtures/multifile/greeting.apib');
-        assert.isObject(dredd.configuration.data['./test/fixtures/multifile/greeting.apib']);
-        assert.property(dredd.configuration.data['./test/fixtures/multifile/greeting.apib'], 'filename');
-        assert.property(dredd.configuration.data['./test/fixtures/multifile/greeting.apib'], 'raw');
+        assert.lengthOf(dredd.configuration.apiDescriptions, 3);
+        dredd.configuration.apiDescriptions.sort((ad1, ad2) => ad1.location.localeCompare(ad2.location));
```

Comment:
There is sorting on multiple places, because Dredd currently resolves the apiDescriptions in an asynchronous way (downloading remote API description files, resolving glob patterns, ...), i.e. in non-deterministic order. To make the resolution deterministic would mean to rewrite half of the `Dredd.js` file, so I opted for sorting the array in tests for now.

Reply:
Is `localeCompare` correct to use here? Perhaps the tests could fail on some users system where they use a locale that is pretty different to our assumptions in the file order as we're expecting the result to be consistent with assertions below.

Reply:
I guess I could use a custom sorting function in that case. It's such a pain to sort in JS 😞 I miss Python's `key` approach.